src/components/FormSelect.js

```diff
@@ -123,10 +123,12 @@ module.exports = React.createClass({
 		return (
 			<div className={componentClass}>
 				{componentLabel}
-				<select className="FormInput FormSelect" id={forAndID} onChange={this.handleChange} onBlur={this.handleBlur} {...props}>
-					{options}
-				</select>
-				{this.renderIcon(icons.selectArrows)}
+				<div className="u-pos-relative">
```

Comment:
Select is inline element, wrapping it in a block element should be a trouble. Maybe span?